src/typings/acsWebhooks/challengeInfo.ts

```diff
@@ -86,7 +86,8 @@ export namespace ChallengeInfo {
         _08 = '08'
     }
     export enum FlowEnum {
-        OtpSms = 'OTP_SMS',
-        Oob = 'OOB'
+        PwdOtpPhoneFl = 'PWD_OTP_PHONE_FL',
```

Comment:
@gcatanese These changes are breaking, but they had the wrong values all along -> https://github.com/Adyen/adyen-dotnet-api-library/pull/1134, let's mention these in the descriptoin